kivy/uix/bubble.py

```diff
@@ -80,6 +80,9 @@ class BubbleButton(Button):
     Rather use this BubbleButton widget that is already defined and provides a
     suitable background for you.
     '''
+    def __init__(self, **kwargs):
+        super(BubbleButton, self).__init__(**kwargs)
+        self.always_release = BooleanProperty(False)
```

Comment:
@akshayaurora sorry for the delay . I realized that the always_release should be a class property and hence changed bubble.py